### PR TITLE
Add Code of Conduct for hackathon

### PR DESCRIPTION
**Pull request by etcadinfinitum**
The elements for this were pulled primarily from https://hackcodeofconduct.org/ with additional notes about plagarism & consequences added by yours truly.

### CODE REVIEW

File: _docs/coc.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* hacks
+* talks, presentations, or demos
+* workshops
+* any parties associated to the hackathon
+* social media
+* any other online media
+
+### Plagarism
+
+We do not tolerate intentional direct plagarism of code or visual content from other participants or readily available resources on the internet.
```
Comment (cforcomputer) [inline _docs/coc.md]:
```suggestion
Intentional direct plagiarism of code or visual content from other participants and readily available resources on the internet will not be tolerated.
```

File: _docs/coc.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+If your project uses visual or programmatic elements that your team didn't create from scratch, you are expected to attribute credit to the original author(s).
```
Comment (cforcomputer) [inline _docs/coc.md]: Simplify language
```suggestion
If your project uses visuals or program elements that your team didn't create from scratch, you are expected to attribute credit to the original author(s).
```

File: _docs/coc.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Please use common sense, and take this opportunity to showcase your own individual creativity.
```
Comment (cforcomputer) [inline _docs/coc.md]:
```suggestion
Please apply common sense in situations not defined in this code of conduct. Take this opportunity to showcase your own individual creativity.
```

File: _docs/coc.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+### Consequences
+
+Hackathon participants violating these rules may be sanctioned at the discretion of the hackathon organisers. Any of the following consequences may result:
+
+* being asked to leave the event
+* being banned from future events hosted by the organizers
+* having awards revoked
+
```
Comment (cforcomputer) [inline _docs/coc.md:66]:
```suggestion
* campus security being notified in the event of direct harassment

```

File: _docs/coc.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+# Code of Conduct
+
+UWB Hacks the Cloud is going to be amazing! Let's lay some ground rules to make sure everyone has the best experience possible.
+
+## TL;DR
+
+The gist of the Code of Conduct is:
+
+* be respectful to each other
+* don't plagarize other people's work
```
Comment (cforcomputer) [inline _docs/coc.md]:
```suggestion
* don't plagiarize other people's work
```

**Review by cforcomputer — CHANGES_REQUESTED** [review]
Made some simple grammar changes.

Looks good overall.

File: _docs/coc.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+* race
+* ethnicity
+* nationality
+* religion
+* political views
+* previous hackathon attendance or lack of
+* computing experience or lack of
+* chosen programming language or tech stack
+
+We do not tolerate harassment of hackathon participants in any form. Sexual language and imagery is not appropriate at any hackathon venue; this includes the following:
```
Comment (tleyler) [inline _docs/coc.md]:
```suggestion
We do not tolerate harassment of hackathon participants, judges, mentors, volunteers, or organizers in any form. Sexual language and imagery is not appropriate at any hackathon venue; this includes the following:
```

File: _docs/coc.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+* hacks
+* talks, presentations, or demos
+* workshops
+* any parties associated to the hackathon
+* social media
+* any other online media
+
+### Plagarism
```
Comment (cforcomputer) [inline _docs/coc.md]:
```suggestion
### Plagiarism 
```

File: _docs/coc.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+Hackathon participants violating these rules may be sanctioned at the discretion of the hackathon organisers. Any of the following consequences may result:
+
+* being asked to leave the event
+* being banned from future events hosted by the organizers
+* having awards revoked
+
+### Inquiries
+
+If you have any questions about these rules, please contact [the hackathon organizers](https://uwbacm.com).
```
Comment (tleyler) [inline _docs/coc.md:69]: A few of the code of conduct examples I just scoped out for inspiration have a way for people to report violations of the code of conduct anonymously which I thought might make it easier for people to speak up if they're feeling uncomfortable, but also might pose a small logistical challenge with organizers having to monitor the form submissions during the hackathon 🤷‍♀  If we want to implement something like that, I can write up a Google Form people can use as a reporting tool and add it here.
```suggestion
If you have any questions about these rules, please contact [the hackathon organizers](https://uwbacm.com).

If you feel uncomfortable or think there may be a potential violation of the code of conduct, please report it immediately using the [Code of Conduct Violation Report]() form. All reporters have the right to remain anonymous.
```

**Review by tleyler — APPROVED** [review]
This looks great! Nice work 😀

File: _docs/coc.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+* being asked to leave the event
+* being banned from future events hosted by the organizers
+* having awards revoked
+* campus security being notified in the event of direct harassment
+
+### Inquiries
+
+If you have any questions about these rules, please contact [the hackathon organizers](https://uwbacm.com).
+
+If you feel uncomfortable or think there may be a potential violation of the code of conduct, please report it immediately using the [Code of Conduct Violation Report]() form. All reporters have the right to remain anonymous.
```
Comment (etcadinfinitum) [inline _docs/coc.md]:
```suggestion
If you feel uncomfortable or think there may be a potential violation of the code of conduct, please report it immediately using the [Code of Conduct Violation Report](https://forms.gle/NXxSczeFCLFaPtBY7) form. All reporters have the right to remain anonymous.
```

**Pull request merged**